app/models/ability.rb

```diff
@@ -49,12 +49,13 @@ def not_signed_in
     # can view Commercials of confirmed Events
     can :show, Commercial, commercialable_type: 'Event', commercialable_id: Event.where(state: 'confirmed').pluck(:id)
     can :show, User
-    can [:show, :create], Registration do |registration|
-      registration.new_record?
-    end
-
-    can [:show, :create], Event do |event|
-      event.new_record?
+    unless CONFIG['authentication']['ichain']['enabled']
+      can [:show, :create], Registration do |registration|
+        registration.new_record?
+      end
+      can [:show, :create], Event do |event|
```

Comment:
[It is](https://github.com/hennevogel/osem/blob/master/app/models/ability.rb#L45) see above